Empire - The Last Battle/Assets/Unity/Scripts/MonoBehaviours/OverworldManager.cs

```diff
@@ -354,17 +366,18 @@ void _StormshapersPlayer_OnCardRemoved(CardData card, int index)
 				// check for total KO
```

Comment:
You need to do the game over/KO check for each battle type. this is just for PvP atm. That's why i suggested doing it first in the function above the switch, or wrapping it overall in a function. and calling it for each case.